docs/site/Sequence.md

```diff
@@ -217,9 +217,44 @@ namespace.
 
 ### Query string parameters
 
-{% include content/tbd.html %}
+OAI 3.0.x describes the data from a request’s header, query and path in an
+operation specification’s parameters property. In a Controller method, such an
+argument is typically decorated by @param(). We've made multiple shortcuts
+available to the `@param()` decorator in the form of
+`@param.<http_source>.<OAI_primitive_type>`. Using this notation, query string
+parameters can be described as `@param.query.<OAI_primitive_type>`. Here is an
+example of a controller method which retrieves a Note model instance by
+obtaining the `id` from the query string object.
+
+```ts
+ @get('/notes/test', {
+    responses: {
+      '200': {
+        description: 'Note model instance',
+        content: {'application/json': {schema: {'x-ts-type': Note}}},
+      },
+    },
+  })
+  async findById(@param.query.string('id') id: string): Promise<Note> {
+    return await this.noteRepository.findById(id);
+  }
+```
+
+You can also specify a parameter which is an object value encoded as a JSON
+string or in multiple nested keys. For a JSON string, a sample value would be
+`location={"lang": 23.414, "lat": -98.1515}`. For the same `location` object, it
+can also be represented as `location[lang]=23.414&location[lat]=-98.1515`. Here
+is the equivalent usage for `@param.query.object()` decorator. It takes in the
+name of the parameter and an optional schema or reference object for it.
+
+```ts
+    @param.query.object('location', {
+      type: 'object',
+      properties: {lat: {type: 'number', format: 'float'}, long: {type: 'number', format: 'float'}},
+    })
+```
 
-How to get query string param values.
+The parameters are retrieved as the result of `parseParams` Sequence action.
 
```

Comment:
The content looks good overall :+1:

Please add a note that deeply nested properties are not officially supported by OAS yet and point people to the tracking issue https://github.com/OAI/OpenAPI-Specification/issues/1706.

It would be also great to mention that because deeply-nested objects are not supported, our REST API Explorer does not allow users to provide values for such parameters and unfortunately has no visible indication of that. This problem is tracked and discussed in https://github.com/swagger-api/swagger-js/issues/1385.